.github/workflows/detect_changes_to_app_job.yml

```diff
@@ -19,16 +19,16 @@ jobs:
         id: detect_changes
         if: github.ref != 'refs/heads/main'
         run: |
-          if [[ "${{github.event.action}}" != "opened" ]]; then
-            echo "Checking against last push"
-            git diff --quiet ${{github.event.before}} ${{github.event.after}} -- app lang web Dockerfile package.json yarn.lock && \
-            echo "changes=${{ false }}" || \
-            echo "changes=${{ true }}"
+          # if [[ "${{github.event.action}}" != "opened" ]]; then
```

Comment:
I've added https://opgtransform.atlassian.net/browse/MLPAB-1022 to make this work